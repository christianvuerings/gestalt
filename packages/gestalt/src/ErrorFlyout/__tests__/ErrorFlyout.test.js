// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import ErrorFlyout from '../ErrorFlyout';

test('ErrorFlyout renders', () => {
  const component = renderer.create(
    <ErrorFlyout anchor={null} message="Email required" onDismiss={() => {}} />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
