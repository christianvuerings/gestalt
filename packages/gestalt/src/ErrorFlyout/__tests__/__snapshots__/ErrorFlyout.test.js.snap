// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorFlyout renders 1`] = `null`;
